modules/session/db.go

```diff
@@ -91,7 +91,7 @@ type DBProvider struct {
 
 // Init initializes DB session provider.
 // connStr: username:password@protocol(address)/dbname?param=value
-func (p *DBProvider) Init(maxLifetime int64, connStr string) error {
+func (p *DBProvider) Init(maxLifetime int64, _ string) error {
```

Comment:
This also looks suspicious given the comment above it.

Reply:
Yeah, the parameter meaning lost.

Reply:
It is the comment that is misleading because `connStr` has no effect in the method behavior (i.e. the parameter means nothing to this method)

Reply:
Yeah the comment is old.